lib/dbi/mongo/dummy.c

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2022 by sysmocom - s.f.m.c. GmbH <info@sysmocom.de>
+ * Author: Alexander Couzens <lynxis@fe80.eu>
+ *
+ * This file is part of Open5GS.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+/* This file will be used when build without mongodb support to support the old api */
+#include "ogs-dbi.h"
+
+int ogs_dbi_init(const char *db_uri)
```

Comment:
why is this needed? This commint looks like needs to be meregd with some of the previous commits doing stuff with the API.

Reply:
Do you know if open5gs cares about so library versions and compatibility?
The dummy functions are needed when building without mongdb. dbi is still a shared library.